app/components/barcode_component.rb

```diff
@@ -34,7 +34,7 @@ def barcode_caption_id
   end
 
   def css_class
-    [*tag_options[:class], 'display-inline-block margin-0']
```

Comment:
This is rendered on a `<div>` element which shouldn't have any inherent margin to be unset.

https://github.com/18F/identity-idp/blob/622f9c52e0456f6e1f178f7aa92c30c51718d5d6/app/components/barcode_component.html.erb#L2-L6